### PR TITLE
#Task_MySQL. Yura Matsiulka

### PR DESCRIPTION
**Pull request by Happy-Love**
https://travis-ci.org/github/Happy-Love/db-assignments/builds/716054171

### CODE REVIEW

File: task/sql-tasks.js [hunk 1]
```diff
@@ -91,7 +124,18 @@ async function task_1_5(db) {
...
 async function task_1_6(db) {
-    throw new Error("Not implemented");
+    // (inner join)
+    let result = await db.query(`
+    SELECT 
+	    p.ProductName as "ProductName",
+        c.CategoryName as "CategoryName",
+        s.CompanyName as "SupplierCompanyName"
+    FROM Products p
+    JOIN Categories c ON p.CategoryID=c.CategoryID
```
Comment (AlexMokin) [inline task/sql-tasks.js]: use **inner join** keyword explicitly

File: task/sql-tasks.js [hunk 2]
```diff
@@ -105,7 +149,16 @@ async function task_1_6(db) {
...
  */
 async function task_1_7(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT 
+	    r.EmployeeID as 'EmployeeId',
+	    CONCAT(r.FirstName,' ',r.LastName) as "FullName",
+	    COALESCE(CONCAT(e.FirstName,' ',e.LastName),'-') as "ReportsTo"
+    FROM Employees e
+    RIGHT OUTER JOIN Employees r ON e.EmployeeID = r.ReportsTo
```
Comment (AlexMokin) [inline task/sql-tasks.js]: Use left outer join if possible

**Review by AlexMokin — COMMENTED** [review]
Looks good. See comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

File: task/sql-tasks.js (hunk 1 above)
Comment (Happy-Love) [reply task/sql-tasks.js]: Okay!

File: task/sql-tasks.js (hunk 2 above)
Comment (Happy-Love) [reply task/sql-tasks.js]: Okay!

**Comment by AlexMokin** [comment]
Please fix code according to the comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by Happy-Love** [comment]
Sorry for this thing that I moved to another branch with the same almost name. I don't know how it happened. But I fixed it

**Comment by AlexMokin** [comment]
Accepting. But what's up with the formatting? Github shows that every line of code has changed. It's very importatnt to be able to see what actually has changed

image

**Comment by Happy-Love** [comment]
I copied the old file from the previous branch and through (ctrl + a) pasted it in the branch which is currently. If anything, I did not delete the previous branch by name, it is the same with the current branch

**Pull request closed without merging**